redis/redis.go

```diff
@@ -63,6 +63,14 @@ func ClientFromConnectionString(
 	if err != nil {
 		return nil, err
 	}
+	// in case connection string was provided in form of host:port
+	// add scheme and parse again
+	if redisurl.Host == "" {
+		redisurl, err = url.Parse("redis://" + connectionString)
```

Comment:
I always say that, sorry for being annoying: could we define "redis://" as const?

Reply:
"redis://" or just "redis"?
asking, because default scheme is "redis", so then it will look like this:
const defaultRedisScheme = "redis" and
url.Parse(defaultRedisScheme + "://" + connectionString)

not sure if it make sense